tests/test_server.py

```diff
@@ -177,15 +177,19 @@ def test_aborting_interceptor(aio):
         assert e.value.details() == "oh no"
 
 
-def test_method_not_found():
+@pytest.mark.parametrize("aio", [False, True])
+def test_method_not_found(aio):
     """Calling undefined endpoints should return Unimplemented.
 
     Interceptors are not invoked when the RPC call is not handled.
     """
-    intr = CountingInterceptor()
+    intr_type = AsyncCountingInterceptor if aio else CountingInterceptor
+    intr = intr_type()
     interceptors = [intr]
```

Comment:
Could this be simplified to:

```python
interceptors = [AsyncCountingInterceptor() if aio else CountiningInterceptor()]
```

Reply:
It’s following the pattern used elsewhere in the tests. I agree it’s weird looking though and I forget why it’s like that. There’s something to be said for both simplifying it and for keeping it consistent. I’m fine either way so will leave it up to you.

Reply:
I have gone ahead and left it as is to be consistent with the other tests. My understanding is that while not many, some of the interceptors accept a parameter in their constructor so this pattern allows passing parameters without repetition. Figured may as well leave them as is, and just stick with it throughout.